pipeline.cpp

```diff
@@ -80,7 +122,37 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
  ************************************************************/
 void DrawTriangle(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* const attrs, Attributes* const uniforms, FragmentShader* const frag)
 {
-    // Your code goes here
+    // define vertices
+    Vertex v1 = triangle[0];
```

Comment:
This code is repetitive as you can just reference the vertices straight from the triangle